site/src/components/form.md

```diff
@@ -216,6 +223,47 @@ If you want to skip the basic validation on an input, apply a `data-skip-validat
 			</label>
 		</div>
 	</div>
+	<div class="field">
+		<legend class="field-label">Radio Field label</legend>
+		<div class="field-body">
+			<label class="radio" for="question-id-1">
+				<input
+					name="question-1"
+					id="question-id-1"
+					type="radio"
+					class="radio-dot"
+					value="Yes"
+					data-validation-label="Radio Input selection"
+					required
+				/>
+				<span class="radio-label-text">Yes</span>
+			</label>
+			<label class="radio" for="question-id-2">
+				<input name="question-1" id="question-id-2" type="radio" class="radio-dot" value="No" />
```

Comment:
super nit: indentation to match other examples?

Reply:
Yep.  On save auto formatting caused the shift here.